src/tests/simulation/src/cluster.rs

```diff
@@ -12,19 +12,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::future::Future;
 use std::io::Write;
-use std::net::IpAddr;
 use std::sync::LazyLock;
 use std::time::Duration;
 
 use anyhow::{bail, Result};
 use clap::Parser;
-use futures::future::BoxFuture;
-use madsim::rand::thread_rng;
+use futures::future::{join_all, BoxFuture};
 use madsim::runtime::{Handle, NodeHandle};
-use rand::seq::SliceRandom;
+use rand::Rng;
+use sqllogictest::AsyncDB;
 
-use crate::RisingWave;
+use crate::client::RisingWave;
 
 /// Embed the config file and create a temporary file at runtime.
 static CONFIG_PATH: LazyLock<tempfile::TempPath> = LazyLock::new(|| {
```

Comment:
Do we want this config for simulation e2e? 🤔

Reply:
The product of these two parameters decides the update frequency of the query results. In scale tests, we rely on the changes in the result to ensure that the scheduling after scaling is covered by the test, so it's better to use a smaller value here.

Reply:
> Is it OK to apply them for all?

I'm not sure, as the time of e2e tests is reduced with larger barrier interval. https://github.com/risingwavelabs/risingwave/pull/6553#issuecomment-1326080734

Reply:
Just changed to allow user to specify a config file from argument for e2e test, and use the default config if not specified.

Reply:
How about only use this config for scale test and allow other tests to use default? The time cost of other simulation test will reduced obviously with larger barrier interval.

Reply:
Yeah, it is this way now.

Reply:
Indeed, that's quite strange that the time cost of recovery test is getting longer, from 4mins+ to 6mins+. 😢